.storybook/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+const appConfig = require('../webpack.common');
+
+module.exports = ({ config }) => {  
+  config.module.rules.push(...appConfig.module.rules);
```

Comment:
Do we need to register `react-docgen-typescript-loader` here?

Reply:
Right, will add.